.gitignore

```diff
@@ -5,8 +5,9 @@ frontend/node_modules/*
 scripts/config/webpack.config.js
 **/.DS_Store
 **/__pycache__/
-backend/uploaded_videos/
+backend/videos/
```

Comment:
I changed the name of the folder to avoid confusion, in rating.py it was "videos" and in main.py it was "uploaded_videos"